coxeter/shape_classes/base_classes.py

```diff
@@ -28,6 +28,38 @@ def gsd_shape_spec(self):
         """dict: Get a :ref:`complete GSD specification <shapes>`."""  # noqa: D401
         return {}
 
+    def compute_form_factor_amplitude(self, q):
+        r"""Calculate the form factor intensity.
+
+        In solid state physics, scattering theory is concerned with understanding the
+        ways in which light is scattered from a sample. For a single point particle at
```

Comment:
Other particles love to scatter too ❤️
```suggestion
        ways in which radiation is scattered from a sample. For a single point particle at
```

Reply:
I thought you were a physicist! From Wikipedia:
> In physics, the term light sometimes refers to electromagnetic radiation of any wavelength, whether visible or not 
but in any case I can change it :)